### PR TITLE
improvement(upgrade_test): create InfoEvent instead of log.info

### PR DESCRIPTION
**Pull request by fgelcer**
while working for few weeks now on upgrade tests,
feels super miserable to find what is happening in
the logs, as we have a well organised `events.log`,
but the "steps" of the upgrade tests are only printed
in the logs, making very hard to follow after errors
for example, as the errors will appear in the events,
but the stage of the test itself won't.
with this fix, it will be all in one place, and it
will allow to follow much easier after where each
error or failure happened.

the task in trello is here:
https://trello.com/c/Ps3ZUcgl/4172-in-rolling-upgrade-test-convert-all-selfloginfo-to-infoevent

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fruch — APPROVED** [review]
LGTM

**Review by soyacz — APPROVED** [review]
LGTM

**Comment by fgelcer** [comment]
@dkropachev , can you please back port it?

**Comment by fgelcer** [comment]
no back port needed, as branch-4.6 was FF'ed
